src/mahoji/lib/abstracted_commands/nightmareCommand.ts

```diff
@@ -133,9 +155,6 @@ export async function nightmareCommand(user: MUser, channelID: string, name: str
 		type = 'mass';
 	}
 
-	const err = await checkReqs([user], NightmareMonster, 2, isPhosani);
```

Comment:
This allows for early code termination/ error messages. qty variable is ultimately reassigned after trip length so checking an single kill would suffice if later error messages about trip length/supplies catch the rest of the conditions